kinder/ci/tools/update-workflows/README.md

```diff
@@ -0,0 +1,64 @@
+## update-workflows
+
+Update-workflows is a tool that can be used to update kinder workflows
+and test-infra Prow jobs.
+
+### Example usage
+
+```shell
+go run ./cmd/main.go --config config.yaml --kubernetes-version v1.21 \
+  --path-test-infra ./test-infra --path-workflows ./workflows \
+  --image-test-infra v20210403-e49d2c6 --skew-size=3
+```
+
+### Flags
+
+- `--config` must point to a static configuration file (see bellow).
+- `--kubernetes-version` is the Kubernetes version that is the base of all the version
```

Comment:
i can rephrase the description, this is in the "the latest kuberentes version to be tested"

we can do detecting of the version in the future, but one problem here is that we currently "switch" this version at arbitrary points in time. if we use the autodetection it can break the CI of pending unrelated PRs that would now require a `update-workflows.sh` PR to merge prior to that.

Reply:
rephrasing should be enough for now, thanks